### PR TITLE
Editorial review: terminology,  clarity, reference consistency

### PR DESCRIPTION
**Pull request by Wind4Greg**
This PR  contains editorial (informative) corrections to  terminology (updates  the  EdDSA terminology based on RFC8032 and FIPS 186-5), small  clarity items (missing words),  and references to items  that have  moved between  referenced specs.

***
<a href="https://pr-preview.s3.amazonaws.com/Wind4Greg/vc-di-eddsa/pull/95.html" title="Last updated on Aug 23, 2024, 9:51 PM UTC (325920d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-eddsa/95/53bfa73...Wind4Greg:325920d.html" title="Last updated on Aug 23, 2024, 9:51 PM UTC (325920d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -267,16 +270,16 @@ <h3>Verification Methods</h3>
           <h4>Multikey</h4>
 
           <p>
-The <a data-cite="VC-DATA-INTEGRITY#multikey">Multikey format</a>, defined in
-[[VC-DATA-INTEGRITY]], is used to express public keys for the cryptographic
+The <a data-cite="controller-document#multikey">Multikey format</a>, defined in
+[[controller-document]], is used to express public keys for the cryptographic
```
Comment (TallTed) [inline index.html]:
```suggestion
[[[controller-document]]], is used to express public keys for the cryptographic
```

File: index.html [hunk 2]
```diff
@@ -267,16 +270,16 @@ <h3>Verification Methods</h3>
...
 suites defined in this specification.
           </p>
 
           <p>
 The `publicKeyMultibase` value of the verification method MUST start with the
 base-58-btc prefix (`z`), as defined in the
-<a data-cite="VC-DATA-INTEGRITY#multibase-0">Multibase section</a> of
-[[VC-DATA-INTEGRITY]]. A Multibase-encoded Multikey value follows, which MUST
+<a data-cite="controller-document#multibase-0">Multibase section</a> of
+[[controller-document]]. A Multibase-encoded Multikey value follows, which MUST
```
Comment (TallTed) [inline index.html]:
```suggestion
[[[controller-document]]]. A Multibase-encoded Multikey value follows, which MUST
```

File: index.html [hunk 3]
```diff
@@ -370,11 +373,11 @@ <h4>DataIntegrityProof</h4>
...
           </p>
           <p>
-The `proofValue` property of the proof MUST be a detached EdDSA
+The `proofValue` property of the proof MUST be a detached EdDSA signature
 produced according to [[RFC8032]], encoded using the base-58-btc header and
 alphabet as described in the
-<a data-cite="VC-DATA-INTEGRITY#multibase-0">Multibase section</a> of
-[[VC-DATA-INTEGRITY]].
+<a data-cite="controller-document#multibase-0">Multibase section</a> of
+[[controller-document]].
```
Comment (TallTed) [inline index.html]:
```suggestion
[[[controller-document]]].
```

File: index.html [hunk 4]
```diff
@@ -409,7 +412,7 @@ <h2>Algorithms</h2>
 
       <p>
 The following section describes multiple Data Integrity cryptographic suites
-that utilize the twisted Edwards Curve Digital Signature Algorithm.
+that utilize the Edwards-Curve Digital Signature Algorithm.
```
Comment (TallTed) [inline index.html]:
```suggestion
that use the Edwards-Curve Digital Signature Algorithm.
```

File: index.html [hunk 5]
```diff
@@ -435,11 +438,11 @@ <h3>Instantiate Cryptosuite</h3>
 If |options|.|cryptosuite| is `eddsa-rdfc-2022`:
             <ol class="algorithm">
               <li>
-Set |cryptosuite|.|createProof| to the result of running the algorithm in Section
+Set |cryptosuite|.|createProof| to the algorithm in Section
```
Comment (TallTed) [inline index.html:441]: I think these changes are incorrect. The algorithm doesn't go into `|cryptosuite|.|createProof|`; its result does. 
```suggestion
Set |cryptosuite|.|createProof| to the algorithm in Section
```

File: index.html [hunk 6]
```diff
@@ -796,9 +799,9 @@ <h4>Proof Verification (eddsa-rdfc-2022)</h4>
 Let `publicKeyBytes` be the result of retrieving the
 public key bytes associated with the
 `options`.`verificationMethod` value as described in the
-Data Integrity [[VC-DATA-INTEGRITY]] specification,
-<a data-cite="vc-data-integrity#algorithms">
-Section 4: Retrieving Cryptographic Material</a>.
+[[controller-document]] specification,
```
Comment (TallTed) [inline index.html]:
```suggestion
[[[controller-document]]] specification,
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (Wind4Greg) [reply index.html:441]: No,  this  is  setting  up the cryptosuite. So  it is setting up the algorithms (functions) to be  called. See  how  this is  done in  https://w3c.github.io/vc-di-ecdsa/#instantiate-cryptosuite.  This is typical of  "pluggable crypto" APIs.

Comment (TallTed) [reply index.html:441]: @Wind4Greg — OK, I see the distinction. Thanks for the pointer and explanation!

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**